dali/test/python/test_operator_laplacian.py

```diff
@@ -0,0 +1,573 @@
+            s_windows[window_size] = s.reshape(-1)
+    for window_size in s_windows:
+        if s_windows[window_size] is None:
+            if window_size == 1:
+                s_windows[window_size] = np.array([1.], dtype=np.float32)
+            else:
+                _, s = cv2.getDerivKernels(2, 0, ksize=window_size)
+                s_windows[window_size] = s.reshape(-1)
+    return [[
+        (d_windows if i == j else s_windows)[window_sizes[i][j]]
+        for j in range(axes)] for i in range(axes)]
+
+
+def get_window_sizes(window_size, smoothing_size, axes):
+    window_sizes = spread_values(window_size, axes)
+    if len(smoothing_size.reshape(-1)) == 0:
+        return [[window_sizes[i]] * axes for i in range(axes)]
+    else:
+        smoothing_sizes = spread_values(smoothing_size, axes)
+        return [[window_sizes[j] if i == j else smoothing_sizes[j] for j in range(axes)] for i in range(axes)]
+
+
+def laplacian_baseline(img, out_type, window_size, smoothing_size, scale, axes, skip_axes=0):
+    scales = spread_values(scale, axes)
+    all_sizes = get_window_sizes(window_size, smoothing_size, axes)
+    acc = np.zeros(img.shape, dtype=np.float32)
+    img = np.float32(img)
+    all_windows = get_windows(all_sizes)
+    for windows, scale in zip(all_windows, scales):
+        partial = img
+        for i in reversed(range(axes)):
+            axis = i + skip_axes
+            if img.shape[axis] == 1:
+                mode = "nearest"
+            else:
+                mode = "mirror"
+            partial = convolve1d(partial, windows[i], axis, mode=mode)
+        acc += scale * partial
+    if out_type == np.float32:
+        return acc
+    else:
+        return convert_sat(acc, out_type)
+
+
+def count_skip_axes(layout):
+    if layout.startswith("FC") or layout.startswith("CF"):
+        return 2
+    if layout.startswith("F") or layout.startswith("C"):
+        return 1
+    return 0
+
+
+@pipeline_def
+def laplacian_per_sample_pipeline(iterator, layout, window_dim, smoothing_dim, axes, normalize, out_type):
+    data = fn.external_source(iterator, layout=layout)
+    if window_dim is None:
+        window_size = 3
+        w_exponent = 0
+        window_arg = None
+    else:
+        window_shape = [axes for _ in range(window_dim)]
+        window_size = fn.random.uniform(
+            range=[1, max_window_size // 2], shape=window_shape)
+        window_size = fn.cast(window_size, dtype=types.INT32) * 2 + 1
+        window_arg = window_size
+        w_exponent = window_size - 3
+
+    if smoothing_dim is None:
+        smoothing_size = None
+        s_exponent = (window_size - 1) * (axes - 1)
+    else:
+        smoothing_shape = [axes for _ in range(smoothing_dim)]
+        smoothing_size = fn.random.uniform(
+            range=[0, max_window_size // 2], shape=smoothing_shape)
+        smoothing_size = fn.cast(smoothing_size, dtype=types.INT32) * 2 + 1
```

Comment:
Shouldn't it be the same?
```suggestion
        smoothing_size = fn.random.uniform(
            range=[0, max_window_size // 2], shape=smoothing_shape, dtype=types.INT32) * 2 + 1
```

Reply:
done